### DISCUSSION TITLE
[request][ratelimiting] Retrieve current rate-limit metrics

### DISCUSSION THREAD

**Discussion by montanaflynn** [Kong/kong#7352] [Ideas and feature requests]
It would be good for both consumers and providers if there was an easy way to get the current rate-limit status though an API that returns JSON. This wouldn't count against any limits imposed by the API provider and be served directly from Kong, never hitting the actual upstream target. 

Here's an example: http://api.ferret-go.com/docs/account.html

**Comment by bvrakvs** [comment]
This would be nice for a custom logic that utilizes the count Kong stores for rate-limiting, making it public wouldn't be too hard. Will this feature ever be realized? Not the best having no update on an issue for 3 whole years!
